src/routes.py

```diff
@@ -164,7 +179,7 @@ def login():
 @app.route("/settings", methods=['GET', 'POST'])
 def settings():
     pynder_session = db_util.load_pynder_session(session['username'])
-
+    profile = Profile(pynder_session._api.profile(), pynder_session._api)
```

Comment:
in this instance, it was a merge error. 
Removed redundant Profile classes from other functions